materials/index.md

```diff
@@ -5,8 +5,144 @@ title: Lecture Materials
 
 ### Agroecology IST
 
+<<<<<<< HEAD
```

Comment:
Remove this line.
Something funky happened with the merge that we aborted during the lab meeting..